fastserde/avro-fastserde-tests-common/src/test/java/com/linkedin/avro/fastserde/logical/types/LogicalTypesFastSerdeGeneratorDummyTest.java

```diff
@@ -0,0 +1,93 @@
+package com.linkedin.avro.fastserde.logical.types;
+
+import static com.linkedin.avro.fastserde.FastSerdeTestsSupport.getCodeGenDirectory;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.net.URLClassLoader;
+
+import org.apache.avro.Schema;
+import org.apache.avro.generic.GenericData;
+import org.testng.annotations.Test;
+
+import com.linkedin.avro.fastserde.FastGenericDeserializerGenerator;
+import com.linkedin.avro.fastserde.FastGenericSerializerGenerator;
+import com.linkedin.avro.fastserde.FastSpecificDeserializerGenerator;
+import com.linkedin.avro.fastserde.FastSpecificSerializerGenerator;
+import com.linkedin.avro.fastserde.generated.avro.FastSerdeLogicalTypesDefined;
+import com.linkedin.avro.fastserde.generated.avro.FastSerdeLogicalTypesTest1;
+import com.linkedin.avro.fastserde.generated.avro.FastSerdeLogicalTypesUndefined;
+
+/**
+ * Temporary test just to generate fast-avro (de)serializer classes
```

Comment:
Why do we generate classes for the unsupported(?) Avro versions? If the generation was limited to only supported versions, then this commit wouldn't contain that much generated code.